geode-core/src/main/java/org/apache/geode/management/internal/json/AbstractJSONFormatter.java

```diff
@@ -94,6 +94,8 @@ public AbstractJSONFormatter(int maxCollectionElements, int serializationDepth,
 
     // register the custom module
     mapper.registerModule(mapperModule);
+    // to support jdk8 java.time if jackson-datatype-jsr310 is included in the classpath
```

Comment:
Can you elaborate a bit why `jackson-datatype-jsr310` is needed? I am not sure why it is necessary to add it.

Reply:
the test shows an example using java.time.LocalTime object. If without this line and without the jackson-datatype-jsr310.jar in the classpath, the java.time.LocalTime will not be formatted correctly.

Reply:
That means if the user uses ` java.time.LocalTime`, it also requires `jackson-datatype-jsr310` to format it properly. However, this dependency is for `integrationTestRuntimeOnly` in `build.gradle`. In `expected-pom.xml`, its scope is `compile`.  We might have to check the open source license requirements for new third party dependency.

Reply:
It's only required for testing. If user's domain object includes that type and they would like it to be formatted correctly, then they will need to include the jar in the classpath (Geode is not shipping with it). We are just calling the `findAndRegisterModuel` in case users have the jar in the classpath.